releases/release-1.32/release-notes/release-notes-draft.json

```diff
@@ -5894,8 +5897,8 @@
   },
   "128387": {
     "commit": "3e3276e9fe14bb35262031d5b03f3456a473a22d",
-    "text": "Label `apps.kubernetes.io/pod-index` added to Pod from StatefulSets is promoted to stable\nLabel `batch.kubernetes.io/job-completion-index` added to Pods from Indexed Jobs is promoted to stable",
-    "markdown": "Label `apps.kubernetes.io/pod-index` added to Pod from StatefulSets is promoted to stable\n  Label `batch.kubernetes.io/job-completion-index` added to Pods from Indexed Jobs is promoted to stable ([#128387](https://github.com/kubernetes/kubernetes/pull/128387), [@alaypatel07](https://github.com/alaypatel07)) [SIG Apps]",
+    "text": "Label `apps.kubernetes.io/pod-index` added to Pods from StatefulSets is promoted to stable. Label `batch.kubernetes.io/job-completion-index` added to Pods from Indexed Jobs is promoted to stable.",
+    "markdown": "Label `apps.kubernetes.io/pod-index` added to Pods from StatefulSets is promoted to stable. Label `batch.kubernetes.io/job-completion-index` added to Pods from Indexed Jobs is promoted to stable. ([#128387](https://github.com/kubernetes/kubernetes/pull/128387), [@alaypatel07](https://github.com/alaypatel07)) [SIG Apps]",
```

Comment:
Labels don't usually get promoted as they are not formally API. Is this right?

Reply:
There's prior mention of this wording in previous release notes. I suggest we keep it and clarify it in parallel, raising a PR if needed, thus not blocking the merge needed for release.